### PR TITLE
IMSV2-3324 - Add menu link/tile to DPS home page

### PR DESCRIPTION
**Pull request by rosariodiaferia-moj**
**Ticket:** https://dsdmoj.atlassian.net/browse/IMSV2-3324
**Description**: This ticket will cover allowing the intelligence management service tile to be available in DPS

- The tile should read ‘Intelligence Management Service’ with 
- The underneath text of the tile should state ‘Manage and view intelligence reports’
- url: https://manage-intelligence.prison.service.justice.gov.uk/

### CODE REVIEW

File: server/services/utils/getServicesForUser.ts [hunk 1]
```diff
@@ -247,9 +247,9 @@ export default (
     },
     {
       id: 'submit-an-intelligence-report',
-      heading: 'Submit an Intelligence Report',
-      description: 'Access to the new Mercury submission form',
-      href: config.serviceUrls.mercurySubmit.url,
+      heading: 'Intelligence Management Service',
+      description: 'Manage and view intelligence reports',
+      href: config.serviceUrls.imsSubmit.url,
       enabled: () => true,
```
Comment (garethmdavies) [inline server/services/utils/getServicesForUser.ts:253]: So this link needs only be shown for users with the IMS User Role. See Pathfinder config for example

File: server/services/utils/getServicesForUser.test.ts [hunk 2]
```diff
@@ -29,6 +29,7 @@ jest.mock('../../config', () => ({
     sendLegalMail: { url: 'url' },
     welcomePeopleIntoPrison: { url: 'url', enabledPrisons: 'LEI,LIV' },
     mercurySubmit: { url: 'url' },
+    imsSubmit: { url: 'url' },
```
Comment (garethmdavies) [inline server/services/utils/getServicesForUser.test.ts:32]: Can we change this to manageIntelligence

File: server/config.ts [hunk 3]
```diff
@@ -161,6 +161,9 @@ export default {
     mercurySubmit: {
       url: get('MERCURY_SUBMIT_URL', 'http://localhost:3001', requiredInProduction),
     },
+    imsSubmit: {
```
Comment (garethmdavies) [inline server/config.ts:164]: Can we change this to manageIntelligence

File: helm_deploy/values-prod.yaml [hunk 4]
```diff
@@ -47,6 +47,7 @@ generic-service:
     SEND_LEGAL_MAIL_URL: https://check-rule39-mail.prison.service.justice.gov.uk
     WELCOME_PEOPLE_INTO_PRISON_URL: https://welcome.prison.service.justice.gov.uk
     MERCURY_SUBMIT_URL: https://submit-a-mercury-report.hmpps.service.justice.gov.uk
+    IMS_SUBMIT_URL: https://manage-intelligence.prison.service.justice.gov.uk
```
Comment (garethmdavies) [inline helm_deploy/values-prod.yaml:50]: Can we change this to MANAGE_INTELLIGENCE_URL

File: helm_deploy/values-preprod.yaml [hunk 5]
```diff
@@ -42,6 +42,7 @@ generic-service:
     SEND_LEGAL_MAIL_URL: https://check-rule39-mail-preprod.prison.service.justice.gov.uk
     WELCOME_PEOPLE_INTO_PRISON_URL: https://welcome-preprod.prison.service.justice.gov.uk
     MERCURY_SUBMIT_URL: https://submit-a-mercury-report-preprod.hmpps.service.justice.gov.uk
+    IMS_SUBMIT_URL: https://manage-intelligence-preprod.prison.service.justice.gov.uk
```
Comment (garethmdavies) [inline helm_deploy/values-preprod.yaml:45]: Can we change this to MANAGE_INTELLIGENCE_URL

File: helm_deploy/values-dev.yaml [hunk 6]
```diff
@@ -44,6 +44,7 @@ generic-service:
     SEND_LEGAL_MAIL_URL: https://check-rule39-mail-dev.prison.service.justice.gov.uk
     WELCOME_PEOPLE_INTO_PRISON_URL: https://welcome-dev.prison.service.justice.gov.uk
     MERCURY_SUBMIT_URL: https://submit-a-mercury-report-dev.hmpps.service.justice.gov.uk
+    IMS_SUBMIT_URL: https://manage-intelligence-dev.prison.service.justice.gov.uk
```
Comment (garethmdavies) [inline helm_deploy/values-dev.yaml:47]: Can we change this to MANAGE_INTELLIGENCE_URL

File: server/services/utils/getServicesForUser.ts (hunk 1 above)
Comment (garethmdavies) [reply server/services/utils/getServicesForUser.ts:253]: Also, just spotted that you have edited the submit Mercury config.. that's not right. You need new config definition for the IMS service link

File: server/services/utils/getServicesForUser.test.ts (hunk 2 above)
Comment (rosariodiaferia-moj) [reply server/services/utils/getServicesForUser.test.ts]: changed to manageIntelligence

File: server/config.ts (hunk 3 above)
Comment (rosariodiaferia-moj) [reply server/config.ts]: changed to manageIntelligence

File: helm_deploy/values-prod.yaml (hunk 4 above)
Comment (rosariodiaferia-moj) [reply helm_deploy/values-prod.yaml]: changed to MANAGE_INTELLIGENCE_URL

File: helm_deploy/values-preprod.yaml (hunk 5 above)
Comment (rosariodiaferia-moj) [reply helm_deploy/values-preprod.yaml]: changed to MANAGE_INTELLIGENCE_URL

File: helm_deploy/values-dev.yaml (hunk 6 above)
Comment (rosariodiaferia-moj) [reply helm_deploy/values-dev.yaml]: changed to MANAGE_INTELLIGENCE_URL

**Review by garethmdavies — DISMISSED** [review]
LGTM

File: server/services/utils/getServicesForUser.test.ts [hunk 7]
```diff
@@ -336,9 +337,14 @@ describe('getServicesForUser', () => {
   })
 
   describe('Submit an Intelligence Report', () => {
```
Comment (garethmdavies) [inline server/services/utils/getServicesForUser.test.ts:339]: Should this be a new test?

**Review by whitfield-mj — APPROVED** [review]

File: server/services/utils/getServicesForUser.test.ts (hunk 7 above)
Comment (rosariodiaferia-moj) [reply server/services/utils/getServicesForUser.test.ts]: New test added for IMS

**Review by garethmdavies — APPROVED** [review]
LGTM

**Review by whitfield-mj — APPROVED** [review]

**Pull request merged**